packages/frontend-api/src/Model/Resolver/Products/ProductsResolver.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Resolver\Products;
+
+use Overblog\GraphQLBundle\Definition\Resolver\AliasedInterface;
+use Overblog\GraphQLBundle\Definition\Resolver\ResolverInterface;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainFacade;
+
+class ProductsResolver implements ResolverInterface, AliasedInterface
+{
+    protected const ARGUMENT_LIMIT = 'limit';
```

Comment:
this is not used

Reply:
Deleted https://github.com/shopsys/shopsys/pull/1471/commits/ddc627175b1151954854f331fd302d149f274259